### ISSUE TITLE
Ideas for more options for the generated code

### ISSUE THREAD

**Issue by simonhenke** [jardicc/alchemist#4]
I usually format the descriptors from

```
 {
      "_obj": "make",
      "_target": [
         {
            "_ref": "layer"
         }
      ],
      "_isCommand": true,
 }
```
to

```
 {
      _obj: 'make',
      _target: {
          _ref: 'layer'
       }

 }
```
While most of this is easily done by auto-formatting it with prettier, it would still be cool to have options to

- Remove quotes from object keys
- Use single quotes for values ('value' instead of "value")
- Remove Array frame from _target when there's only one object in it
- Remove the _isCommand keyword (it doesn't have any use, does it?)

I think those would perfectly fit inside the Code>Options tab.

**Comment by jardicc** [comment]
Note: some descriptor keys like in neural filter might contain `:` character where quotes are required.
Also, this could be possibly helpful: https://www.npmjs.com/package/stringify-object , https://json5.org/

**Comment by jardicc** [comment]
Frame won't be removed. I prefer consistency in this case. I do want to implement other suggestions.

**Comment by jardicc** [comment]
All of these except for frame will be implemented in next version 1.4.0

**Comment by jardicc** [comment]
Published in master branch.
